### PR TITLE
Change text in MissionForm date pickers to display default date value

### LINKED ISSUE #233: Creating a new mission needs to default to today's date
When creating a new mission, although the "Pickup Time" is pre-populated with the current time, the "Select Date" field does not default to the current date ...

Screen Shot 2020-04-14 at 5 25 06 PM

Should default to today's date.

### PR DESCRIPTION
**Pull request by dvalinotti**
resolves #233

Cause of issue was the default value of pickUpDateLabel and dropOffDateLabel being null rather than the current date. State variables that stored the date pickers' values were correctly set to the current date/time of the user. Resolved the issue by setting the pickUpDateLabel and dropOffDateLabel default values to the current date/time stringified.

### CODE REVIEW

File: src/app/page/MakeMission/MissionForm.js [hunk 1]
```diff
@@ -88,15 +92,15 @@ function MissionForm({ handleChange, values, onSubmit, getFile /*, assignHelper,
       return;
     }
     if (stage === "pickUp") {
-      setPickUpDateLabel(date);
+      setPickUpDateLabel(date.toString().substr(0, 15));
```
Comment (qdozaq) [inline src/app/page/MakeMission/MissionForm.js:95]: According to lines `97` and `107` looks like `data` could be undefined so may want to have a fallback on these sets as well

**Comment by azmainamin** [comment]
Merge conflicts

File: src/app/page/MakeMission/MissionForm.js (hunk 1 above)
Comment (dvalinotti) [reply src/app/page/MakeMission/MissionForm.js]: Good catch - Resolved this by adding a condition that checks if `date` is undefined before assigning values

**Comment by dvalinotti** [comment]
> Merge conflicts

Resolved merge conflicts!

**Pull request merged**